hack/install-controller-gen.sh

```diff
@@ -2,18 +2,20 @@
 
 set -ex
 
-CONTROLLER_TOOLS_VERSION=${1:-v0.4.0}
-OUTPUT=${2:-${ORIG_DIR}/bin/controller-gen}
+OUTPUT=bin/controller-gen
 
-WORK_DIR=$(mktemp --tmpdir -d controller-gen-XXX)
-trap cleanup EXIT
+if [ -d "vendor" ]
```

Comment:
This could use a comment to explain that downstream repos tracking metal3 probably vendor dependencies.